src/treasury/treasury-indexer.ts

```diff
@@ -0,0 +1,123 @@
+import { ONE_DAY_MS, ONE_HOUR_MS } from '@badger-dao/sdk';
+import { getDataMapper } from '../aws/dynamodb.utils';
+import { ChartDataBlob } from '../aws/models/chart-data-blob.model';
+import { HistoricTreasurySummarySnapshot } from '../aws/models/historic-treasury-summary-snapshot.model';
+import { toChartDataKey } from '../charts/charts.utils';
+import { ChartTimeFrame } from '../charts/enums/chart-timeframe.enum';
+
+export const TREASURY_NAMESPACE = 'treasury';
+
+export async function indexTreasuryCachedCharts(snapshot: HistoricTreasurySummarySnapshot) {
```

Comment:
so this might move to a utils file or smth since it changed as i wrote it ... 

this ended up being a function - can be generalized to all `ChartDataBlobs` if we tweak the interfaces a bit to figure out 

- what time frame cached charts need updated
- what cached charts need trimmed

this way we can expose an endpoint that queries a known key (fast) and only hits data we know we want to serve (cheap) 🙏🏽